base_geoengine/geo_operators.py

```diff
@@ -132,69 +149,84 @@ def get_rel_field(self, rel_col, rel_model):
            rel search"""
         try:
             rel_model._columns[rel_col]
-        except Exception, exc:
-            raise Exception('Model %s has no column %s' % (rel_model._name, rel_col))
-        return "%s.%s" %(rel_model._table, rel_col)
-
-    def _get_direct_como_op_sql(self, table, col, value, rel_col=None, rel_model=None, op=''):
-        "provide raw sql for geater and lesser operators"
+        except Exception:
+            raise Exception(
```

Comment:
Could a more precice exception be caught?
Could a less generic exception be raised?